tap_google_analytics/__init__.py

```diff
@@ -41,28 +42,30 @@ def discover(config):
     # Generate and return the catalog
     return reports_helper.generate_catalog()
 
+
 def get_selected_streams(catalog):
-    '''
+    """
     Gets selected streams.  Checks for an empty breadcrumb
     and metadata with a 'selected' or an 'inclusion' == automatic entry
-    '''
+    """
     selected_streams = []
     for stream in catalog['streams']:
         stream_metadata = metadata.to_map(stream['metadata'])
 
         # stream metadata will have an empty breadcrumb
         if metadata.get(stream_metadata, (), "selected") \
-          or metadata.get(stream_metadata, (), "inclusion") == 'automatic':
+                or metadata.get(stream_metadata, (), "inclusion") == 'automatic':
```

Comment:
nitpick: Using two kinds of quotes for a string doesn't look good. I understand that the code is legacy, but it would be nice to refactor in places where changes were made to one style (`"`or `'`). WDYT?